earth_enterprise/src/fusion/autoingest/plugins.src/PacketGen.src

```diff
@@ -45,6 +45,7 @@ class PacketGenExtraUpdateArg {
 #include <autoingest/plugins/RasterProductAsset.h>
 #include <autoingest/plugins/MercatorRasterProductAsset.h>
 #include <chrono>
+#include <typeinfo>
```

Comment:
Is this still needed? I don't see anywhere it's used, but I could have missed something.